src/dispatcher.js

```diff
@@ -220,14 +193,12 @@ function dispatch (inEvent) {
 * @private
 */
 dispatcher.bubble = function(inEvent) {
-	// '|| window.event' clause needed for IE8
-	var e = inEvent || global.event;
-	if (e) {
-		// We depend on e.target existing for event tracking and dispatching.
-		if (!e.target) {
-			e.target = e.srcElement;
+	if (inEvent) {
+		// We depend on inEvent.target existing for event tracking and dispatching.
+		if (!inEvent.target) {
```

Comment:
I think we can remove this entirely as it was IE-specific and IE9+ supports `Event.target`.